.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,266 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'mkdocs.yml'
+      - '**.md'
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
+      - 'refs/pull/*'
+    paths-ignore:
+      - 'mkdocs.yml'
+      - '**.md'
+
+env:
+  COMPOSER_ARGS_DEFAULT: --ansi --no-interaction --no-progress --prefer-dist
+  COMPOSER_ARGS_PHP8: --ansi --no-interaction --no-progress --prefer-dist --ignore-platform-req=php
+  CONFIG_FILE: .github/workflows/continuous-integration-config.json
+  PHP_DEFAULT_EXTENSIONS: '["mbstring","intl","readline"]'
+  PHP_DEFAULT_INI: '["memory_limit=-1"]'
+
+jobs:
+  lint-json:
+    name: Lint JSON files
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: json-syntax-check
+        run: |
+          echo "Linting composer.json"
+          jq empty composer.json
```

Comment:
Probably need to lint `composer.lock`, when existing

Reply:
There is also  `composer validate` which verifies if `composer.lock` is in sync with `composer.json`.
But do we really need `composer.lock` in any of our packages?

Reply:
> But do we really need `composer.lock` in any of our packages?

If you want to run phpstan, psalm or phpcs, you really need a `composer.lock`, otherwise it will be quite the fun ride :|